quesma/quesma/ui/console_routes.go

```diff
@@ -107,6 +107,58 @@ func (qmc *QuesmaManagementConsole) createRouting() *mux.Router {
 		_, _ = writer.Write(buf)
 	})
 
+	checkIfAbAvailable := func(writer http.ResponseWriter, req *http.Request) bool {
+		if qmc.hasABTestingTable() {
+			return true
+		}
+
+		_, _ = writer.Write([]byte("AB Testing results are not available."))
+		return false
+	}
+
+	authenticatedRoutes.HandleFunc("/ab-testing-dashboard", func(writer http.ResponseWriter, req *http.Request) {
```

Comment:
nitpick: this `/ab-testing-dashboard/` string is used in few places. It would make sense to extract it as const or common var